### PR TITLE
feat: Improve node single connected to more closely mimic ContentSingle. 

### PR DESCRIPTION
**Pull request by vinnyjth**
## TODO:

- [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] Closes [tag-issues-here]
- [ ] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android
- [ ] Set a relevant reviewer

## REVIEW:

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

**Comment by vinnyjth** [comment]
Waiting on #1658

**Comment by vinnyjth** [comment]
@redreceipt There's a companion PR for this. I'll also upload some screenshots when I get a chance.

**Comment by redreceipt** [comment]
left a couple of comments in the companion PR, something isn't quite right for me

File: packages/apollos-ui-connected/src/NodeSingleConnected/index.js [hunk 1]
```diff
@@ -36,7 +38,7 @@ NodeSingleInner.propTypes = {
   ImageWrapperComponent: PropTypes.any, // eslint-disable-line
 };
 
-const NodeSingleConnected = ({ nodeId }) => (
+const NodeSingleConnected = ({ nodeId, children }) => (
```
Comment (redreceipt) [inline packages/apollos-ui-connected/src/NodeSingleConnected/index.js:41]: Pr looks great, just curious, why did we need to add `children` here?

**Review by redreceipt — APPROVED** [review]

File: packages/apollos-ui-connected/src/NodeSingleConnected/index.js (hunk 1 above)
Comment (vinnyjth) [reply packages/apollos-ui-connected/src/NodeSingleConnected/index.js:41]: Added so we could pass in custom footer content, like the `ActionBar`  (which will likely be nixed soon)

**Pull request merged**